LayoutTests/fast/preloader/image-in-nested-template.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<body>
+    <script>
+    if (window.testRunner) {
+        testRunner.dumpAsText();
+        testRunner.dumpResourceResponseMIMETypes();
+    }
+    </script>
+    This test requires DumpRenderTree to see the log of what resources are loaded.
```

Comment:
We can also wait for load event then query performance timeline.
See https://github.com/web-platform-tests/wpt/blob/master/preload/modulepreload.html for an example.

Reply:
@rniwa - Is it for learning or should I look into updating test?